### PR TITLE
Added LCE packages and puppet modules views/helpers

### LINKED ISSUE #123: Views: Lifecycle environment entity
Location:
Content -> Lifecycle Environments

Views:
* [x] Lifecycle environments
* [x] New environment
* [x] LCE Details - Details tab
* [ ] LCE Details - Packages tab
* [ ] LCE Details - Puppet Modules tab

### PR DESCRIPTION
**Pull request by abalakh**
Closes #123

### CODE REVIEW

File: airgun/entities/lifecycleenvironment.py [hunk 1]
```diff
@@ -34,14 +34,28 @@ def create(self, values, prior_entity_name=None,):
...
+        lifecycle environments page
+        """
         view = self.navigate_to(self, 'All')
         return view.read()
 
     def update(self, values, entity_name='Library'):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         return view.fill(values)
 
+    def search_package(self, entity_name, package_name, cv_name=None):
```
Comment (oshtaier) [inline airgun/entities/lifecycleenvironment.py]: `, repo_name`?

File: airgun/entities/lifecycleenvironment.py [hunk 2]
```diff
@@ -34,14 +34,28 @@ def create(self, values, prior_entity_name=None,):
...
+        """Search for specific package inside lifecycle environment"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.packages.search(package_name, cv=cv_name)
```
Comment (oshtaier) [inline airgun/entities/lifecycleenvironment.py]: and here

File: airgun/entities/lifecycleenvironment.py [hunk 3]
```diff
@@ -34,14 +34,28 @@ def create(self, values, prior_entity_name=None,):
...
+        return view.packages.table.read()
+
```
Comment (oshtaier) [inline airgun/entities/lifecycleenvironment.py]: add `search_puppet_module`? or we do not need it?

Comment (abalakh) [reply airgun/entities/lifecycleenvironment.py]: we don't need `search_*` methods to fetch associated packages/puppet_modules/etc, `read()` returns them. All the remaining t2/t3 tests are fetching what's associated, there's no need in search_* methods.
`search_package` is rather exclusion as there is customer's defect which affected searching of packages so separate tests which target package searching were added.

**Comment by oshtaier** [comment]
We can add search functionality in future if we needed them

**Pull request merged**